lib/archive.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const fsPromises = require('fs').promises;
+
+const NODE_ENV = process.env.NODE_ENV;
+const { mArchiveFile, mgArchiveFile, MgProcess } = require('../config.js');
+
+class MArchive {
+  static put(args) {
+    return fsPromises.writeFile(mArchiveFile, JSON.stringify(args));
```

Comment:
Perhaps encapsulate fsPromises in a function with a name that explains what is happening, e.g., `func write_json_to_archive_file`

Reply:
This class method is currently be consumed as: `MArchive.put`, perhaps we could go down the lines of having functions `MArchive.write_json` and `MArchive.read_json`?

I think consuming this method as: `MArchive.write_json_to_archive_file` feels a little bit too long. Perhaps I could change the module name?

But I think when I'm looking to use this method, it's not important that it's written to a file (we could update the implementation to use something else). As long as it's written somewhere and can be read from that same place is all that matters.

Reply:
I agree with your suggested names. Good point about the function not necessarily writing to a file.